app/controllers/api/v1/hosts_controller.rb

```diff
@@ -40,7 +41,8 @@ def show
         param :operatingsystem_id, String, :desc => "required if host is managed and value is not inherited from host group"
         param :medium_id, String, :desc => "required if not imaged based provisioning and host is managed and value is not inherited from host group"
         param :ptable_id, :number, :desc => "required if host is managed and custom partition has not been defined"
-        param :subnet_id, :number, :desc => "required if host is managed and value is not inherited from host group"
+        param :subnet_id, :number, :desc => "IPv4 subnet, required if host is managed and value is not inherited from host group"
+        param :subnet6_id, :number, :desc => "IPv6 subnet"
```

Comment:
The IPv6 Subnet should be optional. Same goes for the IPv4 subnet.
Of course, you can't provision a host without any network configuration. But I think there is no reason why the creation should't be allowed via API.

```
2.1.3 :002 > h = Host::Managed.new
2.1.3 :005 > h.name = 'kamikazehost'
 => "kamikazehost"
2.1.3 :006 > h.valid?
 => true
2.1.3 :009 > h.primary_interface.valid?
 => true
```

@domcleal : Am I missing something?